lib/invoice_generator.rb

```diff
@@ -79,6 +89,8 @@ def run
 
           invoice = Invoice.create_with_id(project_id: project.id, invoice_number: invoice_number, content: project_content, begin_time: @begin_time, end_time: @end_time)
 
+          # Don't substract the 1$ credit from customer's overall credit as it will be applied each month to each customer
+          project_content[:credit] -= project_content.fetch(:github_credit, 0)
```

Comment:
When we add `github_credit` to `credit`, it appears on the current usage page. However, after finalizing the invoice, it disappears from the invoice. It will be something like:
```
subtotal: $80
total: $79
```

IMHO, we shouldn't subtract the credit when saving the result from the `project_content[:credit]`. You might consider altering the `project.credit = Sequel[:credit] - project_content[:credit]` line to avoid deducting $1 credit from the customer's total credit.